docs/vscode-online.md

```diff
@@ -0,0 +1,31 @@
+# VSCode online
+
+It is possible to host an instance of Visual Studio Code in the cloud and have an URL with all your files (protectd with a password), being able to use it in any browser and device.
+
+## Foam with DigitalOcean
+
+- Requirements: a DigitalOcean account and Foam hosted in git (GitHub, Azure DevOps, etc). Extra: Needed a domain and a certificate to have a url like: https://foam.yourdomain.com/
+
+### Create the droplet
+- Go to: Droplets, push button: "Create Droplet" and over Choose and image go to marketplace and select: "code-server"
+- Next choose plan: Basic, 10/mo (I think the 5/mo plan can be not enough)
```

Comment:
```suggestion
- Next choose plan: Basic, 10/mo (I think the 5/mo plan might not be enough)
```

Reply:
Thanks for your comment. I'm using the 5/mo version right now and it looks pretty good.